src/main/webapp/script.js

```diff
@@ -22,8 +22,8 @@ function listElections() {
       let electionList = JSON.parse(textResponse).elections;
       // ocdDivisionId is in the form "ocd-division/country:us/state:<state_code>"
       // For example, Lousiana's ID is "ocd-division/country:us/state:la"
-      let stateRegex = /ocd\-division\/country\:us\/state\:[a-z][a-z]/;
-      let districtRegex = /ocd\-division\/country\:us\/district\:[a-z][a-z]/;
+      let stateRegex = /ocd\-division\/country\:us\/state\:([a-z][a-z])/;
```

Comment:
\* Just FYI, no action required \*

A slight optimization is using [regex alternation](https://www.regular-expressions.info/alternation.html)
You can take advantage of the fact that the two regex is almost identical except for the fact that one uses "state" and the other uses "district".  So if we use this regex:
`/ocd\-division\/country\:us\/(state|district)\:([a-z][a-z])/`
** Note that I added an extra capture group surrounding (state|district) so that the alternate as a whole is treated as one entity.  You can try experimenting what happens when you don't have the parentheses.

It will match for both "ocd-division/country:us/state:la". and "ocd-division/country:us/district:dc"

For example: https://regex101.com/r/KlNoFT/1/tests [Regex101 is a great tool for testing your your regex, and for explaining what's going on in the regex]

Reply:
Ohh thank you!